azurelinuxagent/common/event.py

```diff
@@ -225,6 +227,20 @@ def add_event(self,
         except EventError as e:
             logger.error("{0}", e)
 
+    def add_log_event(self, level, message):
+        event = TelemetryEvent(7, "FFF0196F-EE4C-4EAF-9AA5-776F622DEB4F")
+        event.parameters.append(TelemetryEventParam('EventName', WALAEventOperation.Log))
+        event.parameters.append(TelemetryEventParam('CapabilityUsed', logger.LogLevel.STRINGS[level]))
+        event.parameters.append(TelemetryEventParam('Context1', ''))
```

Comment:
This should be the time in UTC, to ensure we properly record the value.  The timestamp of the event may differ from the timestamp of when it happened.